ARCs/arc-0034.md

```diff
@@ -0,0 +1,86 @@
+---
+arc: 34
+title: Governance proposal process
+description: Criteria for the creation of proposals.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+The goal of this proposal is to increase transparency and efficiency in the decision-making process by clearly defining the steps involved and ensuring that all proposals are given proper consideration. 
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+
+### What is a proposal
+A proposal within the xGov program aims to provide funding for:
+- Funding for early stage projects
+- Facilitation of community involvement
+- Addressing the needs of the community
+ 
+### Duty of a proposer
+Having the ability to propose measures for a vote is a significant power, which requires:
+- A thorough understanding of the needs of the community
+- Alignment of personal interests with the advancement of the Algorand ecosystem.
+> Issues requiring resolution may have been discussed on various online platforms such as forums, discord, and social media networks.
+
+### Voting System
+The voting system for the xGov program will be the following:
+- Only members of xGov are eligible to participate in the voting process
+- Utilization of the Algo committed by xGov as a form of voting currency, with a 1:1 ratio of Algos to votes
+- Proposals that do not meet the minimum threshold of votes required will be removed from consideration.
+
+### Threshold 
+In order for a proposal to be approved, it is necessary for the number of votes in favor of the proposal to be proportionate to the amount of funds requested. This ensures that the allocation of funds is in line with the community's consensus and in accordance with democratic principles.
+
+> eg. 2 000 000 Algo are available to be given away as grants, 200 000 000 millions Algo are committed to the xGov Process (200 000 000 votes available):
```

Comment:
There is potential for spam proposals when we only consider Algo requested. We should have two vote totals. First one is for approval based on Algos (200 000 000 votes total split between all proposals described below), and the second one being approval based on overall sentiment (200 000 000 votes PER Proposal) with a 30% threshold to advance

Reply:
The second vote would allow us to approve/ignore proposals that are not grant-related .

Reply:
If a proposal is not grant-related it probably won't meet the verification criteria upon submission. 
@SudoWeezy could you confirm that please?